apps/surfer/public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="%REACT_APP_SURFER_GUIDELINES_URL%"></script>
```

Comment:
We should definitely make an npm package and not redeclare those types. Could you create a ticket for that?

Reply:
Yes, I'd like to do it after submitting this PR to Contentful review though. I'll create a ticket for it.